### PR TITLE
change detectionlists to use v2 api url

### PR DESCRIPTION
**Pull request by kiran-chaudhary**
### Description of Change ###
It uses the  "employee case management" v2 endpoint to fetch micro-service url.

### Issues Resolved ###
- closes #1199

### Testing Procedure ###

To verify all method under `sdk.detectionlists.`
e.g.
```
sdk.detectionlists.get_user(9999)
sdk.detectionlists.departemployee.get_all()
sdk.detectionlists.highriskemployee.get_all()
```

### PR Checklist ###
Did you remember to do the below?

- [Y ] Add unit tests to verify this change
- [Y] Add an entry to CHANGELOG.md describing this change
- [ NA] Add docstrings for any new public parameters / methods / classes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ how a consumer would use the library (e.g. adding unit tests, updating documenta
 
 - Bug where an empty destination list in a device's backup set broke creation of DeviceSettings objects for that device.
 
+### Changed
```
Comment (unparalleled-js) [inline CHANGELOG.md:17]: IT was actually still managing to use the v2 one before, so this does not need a CHANGELOG entry

Comment (kiran-chaudhary) [reply CHANGELOG.md]: removed.

File: src/py42/services/detectionlists/user_profile.py [hunk 2]
```diff
@@ -9,17 +9,15 @@ class DetectionListUserService(BaseService):
...
+    _resource = u"v2/user"
 
     def __init__(self, connection, user_context, user_service):
         super(DetectionListUserService, self).__init__(connection)
         self._user_context = user_context
         self._user_service = user_service
 
     def _make_uri(self, action):
-        return u"{}{}{}".format(self._uri_prefix, self._resource, action)
+        return u"{}{}".format(self._resource, action)
```
Comment (unparalleled-js) [inline src/py42/services/detectionlists/user_profile.py:20]: It is a tad strange that we have the methods include the slash when calling this helper method. Not a big deal, but just odd.

**Review by unparalleled-js — APPROVED** [review]
Saw 1 slight improvement we could make but not terribly important at all - looks good!

**Pull request merged**